extension/persistence/eclipselink/src/test/java/com/snowflake/polaris/persistence/impl/eclipselink/PolarisEclipseLinkMetaStoreTest.java

```diff
@@ -36,11 +43,20 @@ public class PolarisEclipseLinkMetaStoreTest extends PolarisMetaStoreManagerTest
 
   @Override
   protected PolarisTestMetaStoreManager createPolarisTestMetaStoreManager() {
+    Path tmpFile = Paths.get(String.format("/tmp/%s-persistence.xml", UUID.randomUUID()));
+    try {
+      Files.copy(
+          Objects.requireNonNull(getClass().getResourceAsStream("/META-INF/persistence.xml")),
+          tmpFile);
+    } catch (IOException e) {
+      throw new RuntimeException(e);
+    }
+
     PolarisDiagnostics diagServices = new PolarisDefaultDiagServiceImpl();
     PolarisEclipseLinkStore store = new PolarisEclipseLinkStore(diagServices);
     PolarisEclipseLinkMetaStoreSessionImpl session =
         new PolarisEclipseLinkMetaStoreSessionImpl(
-            store, Mockito.mock(), () -> "realm", null, "polaris-dev");
+            store, Mockito.mock(), () -> "realm", tmpFile.toString(), "polaris-dev");
```

Comment:
For this, Persistence.createEntityManagerFactory() will use the one in the jar, not the /tmp/%s-persistence.xml.

Reply:
You can try to remove persistence.xml from the project and see the issue.